kubectl-minio/cmd/tenant-report.go

```diff
@@ -124,6 +124,13 @@ func (d *reportCmd) run(args []string) error {
 		return err
 	}
 	zipw := zip.NewWriter(w)
+	specAsYaml, err := yaml.Marshal(tenant.Spec)
+	if err == nil {
+		f, err := zipw.Create(tenant.Name + ".yaml")
+		if err == nil {
+			f.Write(specAsYaml)
+		}
+	}
```

Comment:
shouldn't it be entire `tnenat` yaml instead of just `spec` ? @adfost

Reply:
ok I can change that, wasn't sure what was desired